modules/hosted-control-planes-version-support.adoc

```diff
@@ -28,11 +31,8 @@ The HyperShift Operator manages the lifecycle of hosted clusters that are repres
       namespace: hypershift
 ----
 
-The CLI is a helper utility for development purposes. The CLI is released as part of any HyperShift Operator release. No compatibility policies are guaranteed.
+The `hcp` CLI is a getting started tool to create hosted clusters.
```

Comment:
```suggestion
The `hcp` CLI is a getting-started tool to create hosted clusters.
```
Used this way, we need a hyphen. Gonna merge anyway, but if you come back to this section at some point, you can fix. See https://owl.purdue.edu/owl/general_writing/punctuation/hyphen_use.html

Reply:
Addressed.

Reply:
https://github.com/openshift/openshift-docs/pull/76045